mem3_helper.py

```diff
@@ -49,6 +50,10 @@ def discover_peers(service_record):
         print('Discovered', peers_count, 'of', expected_peers_count, 'peers:', peers)
         if peers_count != expected_peers_count:
             print('Waiting for cluster DNS to fully propagate...')
+            statefulset_name = re.search(r'(\w+-\w+-)\d+\.(.*)',peers[0])
+            expected_peers = ['%s%s.%s'%(statefulset_name[1], peer_num, statefulset_name[2]) for peer_num in list(range(expected_peers_count))]
+            for peer_not_found in [peer for peer in expected_peers if peer not in peers]:
+                print('ERROR: Missing couchdb node in the cluster: %s'%(peer_not_found), file=sys.stderr)
```

Comment:
I don't think it should be throwing errors while waiting for the cluster to converge.

Reply:
Solved